examples/reference_environments/gcs/jupyter.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM jupyter/minimal-notebook:python-3.10
+
+WORKDIR /gx
+
+COPY gcs_public_nyc_tlc_bucket_example.ipynb ./
+
+RUN pip install 'great_expectations[bigquery, sqlalchemy-less-than-2]'
```

Comment:
Mind just adding a note here that the sqlalchemy-less-than-2 is temporary (similar to the bigquery one?)

Reply:
Added the same note as the `bigquery` Reference Environment, with the caveat that I added a point to this week's Arch Review to discuss how we might be able to split up `bigquery` and `gcs` requirements. :)